types/node/v11/globals.d.ts

```diff
@@ -1081,21 +1081,25 @@ declare namespace NodeJS {
     }
 
     interface Timer {
+        hasRef(): boolean;
         ref(): void;
         refresh(): void;
```

Comment:
Shouldn't these APIs here also return `this`?

Reply:
Yes, of course. I'm surprised that test didn't catch that.

Reply:
Most likely because your test uses `class Timeout` which was adapted already. I find it interesting that typescript allows a class to implement an interface where return type of methods differs. Maybe because interface uses `void` so the nothing get's broken if interface api is used.